lib/comma_splice/file_corrector.rb

```diff
@@ -64,12 +64,15 @@ def to_json
 
     def line_correctors
       @line_correctors ||= csv_content.collect do |line|
-        LineCorrector.new(header, Line.new(line), @start_column, @end_column)
+        LineCorrector.new(header, Line.new(line, separator), @start_column, @end_column, separator)
       end
     end
 
     def corrected_lines
       line_correctors.collect do |line|
+        # if line.original.include?('http://www.affaritaliani.it/economia/tim-fulvio-conti-resta-smentita-ipotesi-di-passaggio-a-open-fiber-597756.html')
```

Comment:
I think you accidentally checked this in.

Reply:
you're right.